LayoutTests/imported/w3c/web-platform-tests/service-workers/service-worker/fetch-mixed-content-to-outscope.https-expected.txt

```diff
@@ -1,4 +1,4 @@
 
 
-PASS Verify Mixed content of fetch() in a Service Worker
+FAIL Verify Mixed content of fetch() in a Service Worker assert_equals: expected "finish" but got "FAIL(3)FAIL(4)finish"
```

Comment:
I think our behavior is correct according to Fetch because upgrading the request happens in Main Fetch before the service worker is consulted in HTTP Fetch.